### ISSUE TITLE
Errno 10061] No connection could be made

### ISSUE THREAD

**Issue by aerorush91** [evilhero/mylar#1645]
2017-05-31 20:45:04	ERROR	Uncaught exception: Traceback (most recent call last):
File "c:\Mylar\mylar\logger.py", line 158, in new_run
old_run(*args, **kwargs)
File "c:\Python27\lib\threading.py", line 763, in run
self.__target(*self.__args, **self.__kwargs)
File "c:\Mylar\mylar\webserve.py", line 1460, in queueissue
foundcom, prov = search.search_init(ComicName, ComicIssue, ComicYear, SeriesYear, Publisher, issues['IssueDate'], storedate, IssueID, AlternateSearch, UseAFuzzy, ComicVersion, mode=mode, ComicID=ComicID, manualsearch=manualsearch, filesafe=ComicName_Filesafe, allow_packs=AllowPacks)
File "c:\Mylar\mylar\search.py", line 260, in search_init
findit = NZB_SEARCH(ComicName, IssueNumber, ComicYear, SeriesYear, Publisher, IssueDate, StoreDate, searchprov, send_prov_count, IssDateFix, IssueID, UseFuzzy, newznab_host, ComicVersion=ComicVersion, SARC=SARC, IssueArcID=IssueArcID, RSS="no", ComicID=ComicID, issuetitle=issuetitle, unaltered_ComicName=unaltered_ComicName, allow_packs=allow_packs)
File "c:\Mylar\mylar\search.py", line 1609, in NZB_SEARCH
searchresult = searcher(nzbprov, nzbname, comicinfo, entry['link'], IssueID, ComicID, tmpprov, newznab=newznab_host)
File "c:\Mylar\mylar\search.py", line 2350, in searcher
send_to_nzbget = server.append(nzbpath, str(mylar.NZBGET_CATEGORY), int(nzbgetpriority), True, nzbcontent64)
File "c:\Python27\lib\xmlrpclib.py", line 1233, in __call__
return self.__send(self.__name, args)
File "c:\Python27\lib\xmlrpclib.py", line 1591, in __request
verbose=self.__verbose
File "c:\Python27\lib\xmlrpclib.py", line 1273, in request
return self.single_request(host, handler, request_body, verbose)
File "c:\Python27\lib\xmlrpclib.py", line 1301, in single_request
self.send_content(h, request_body)
File "c:\Python27\lib\xmlrpclib.py", line 1448, in send_content
connection.endheaders(request_body)
File "c:\Python27\lib\httplib.py", line 997, in endheaders
self._send_output(message_body)
File "c:\Python27\lib\httplib.py", line 850, in _send_output
self.send(msg)
File "c:\Python27\lib\httplib.py", line 812, in send
self.connect()
File "c:\Python27\lib\httplib.py", line 793, in connect
self.timeout, self.source_address)
File "c:\Python27\lib\socket.py", line 571, in create_connection
raise err
error: [Errno 10061] No connection could be made because the target machine actively refused it

Not sure what is happening here just started doing this on manual search.

**Comment by evilhero** [comment]
It would seem that mylar can't connect to your nzbget instance - is it running and are the values still correct within mylar? 

Did you upgrade nzbget recently?

**Comment by aerorush91** [comment]
No nzbget hasn't been upgraded nothing has changed to nzb since I first installed it. I have Sonarr running through nzb and the settings are pretty much identical.

**Comment by evilhero** [comment]
What are the settings you have within mylar for your nzbget? It's saying it can't connect to your nzbget to send the nzb.. Not sure if there's any additional logs if you put it into debug and send, but you might want to try that and paste the log here

**Comment by milkplus27** [comment]
I'm getting the same error msg [Error 10061] it will download the file but then error out when attempting to copy it over to my NAS. 

The problem happened both before and after I updated SABnzbd to the latest version

**Comment by aerorush91** [comment]
Here is the debug mode:
2017-06-03 08:48:53	ERROR	Uncaught exception: Traceback (most recent call last):
File "c:\Mylar\mylar\logger.py", line 158, in new_run
old_run(*args, **kwargs)
File "c:\Python27\lib\threading.py", line 763, in run
self.__target(*self.__args, **self.__kwargs)
File "c:\Mylar\mylar\webserve.py", line 1460, in queueissue
foundcom, prov = search.search_init(ComicName, ComicIssue, ComicYear, SeriesYear, Publisher, issues['IssueDate'], storedate, IssueID, AlternateSearch, UseAFuzzy, ComicVersion, mode=mode, ComicID=ComicID, manualsearch=manualsearch, filesafe=ComicName_Filesafe, allow_packs=AllowPacks)
File "c:\Mylar\mylar\search.py", line 260, in search_init
findit = NZB_SEARCH(ComicName, IssueNumber, ComicYear, SeriesYear, Publisher, IssueDate, StoreDate, searchprov, send_prov_count, IssDateFix, IssueID, UseFuzzy, newznab_host, ComicVersion=ComicVersion, SARC=SARC, IssueArcID=IssueArcID, RSS="no", ComicID=ComicID, issuetitle=issuetitle, unaltered_ComicName=unaltered_ComicName, allow_packs=allow_packs)
File "c:\Mylar\mylar\search.py", line 1609, in NZB_SEARCH
searchresult = searcher(nzbprov, nzbname, comicinfo, entry['link'], IssueID, ComicID, tmpprov, newznab=newznab_host)
File "c:\Mylar\mylar\search.py", line 2350, in searcher
send_to_nzbget = server.append(nzbpath, str(mylar.NZBGET_CATEGORY), int(nzbgetpriority), True, nzbcontent64)
File "c:\Python27\lib\xmlrpclib.py", line 1233, in __call__
return self.__send(self.__name, args)
File "c:\Python27\lib\xmlrpclib.py", line 1591, in __request
verbose=self.__verbose
File "c:\Python27\lib\xmlrpclib.py", line 1273, in request
return self.single_request(host, handler, request_body, verbose)
File "c:\Python27\lib\xmlrpclib.py", line 1301, in single_request
self.send_content(h, request_body)
File "c:\Python27\lib\xmlrpclib.py", line 1448, in send_content
connection.endheaders(request_body)
File "c:\Python27\lib\httplib.py", line 997, in endheaders
self._send_output(message_body)
File "c:\Python27\lib\httplib.py", line 850, in _send_output
self.send(msg)
File "c:\Python27\lib\httplib.py", line 812, in send
self.connect()
File "c:\Python27\lib\httplib.py", line 793, in connect
self.timeout, self.source_address)
File "c:\Python27\lib\socket.py", line 571, in create_connection
raise err
error: [Errno 10061] No connection could be made because the target machine actively refused it
2017-06-03 08:48:51	DEBUG	Cache Directory successfully found at : c:\Mylar\cache
2017-06-03 08:48:51	DEBUG	[FILENAME] end nzbname: Infinite.Seven.004.2017.digital.dargh-Empire.nzb
2017-06-03 08:48:51	DEBUG	[FILENAME] nzbname (\s): Infinite.Seven.004.2017.digital.dargh-Empire.nzb
2017-06-03 08:48:51	DEBUG	[FILENAME] filename (remove chars): Infinite.Seven.004.2017.digital.dargh-Empire.nzb
2017-06-03 08:48:50	INFO	Download URL: https://dognzb.cr/fetch/c2b144020030b71f711819cac49cbad2/d13f29b3bc08973f3b97f878923f1259 [VerifySSL:True]
2017-06-03 08:48:50	DEBUG	link given by: dognzb
2017-06-03 08:48:50	INFO	Found Infinite Seven (2017) issue: 4 using dognzb
2017-06-03 08:48:50	DEBUG	issues match!
2017-06-03 08:48:50	DEBUG	nzbname used for post-processing:Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 08:48:50	DEBUG	[SEARCHER] end nzbname: Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 08:48:50	DEBUG	[SEARCHER] nzbname (\s): Infinite Seven 004 2017 digital dargh-Empire
2017-06-03 08:48:50	DEBUG	[SEARCHER] nzbname (remove chars): Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 08:48:50	DEBUG	[SEARCHER] entry[title]: Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 08:48:50	DEBUG	integer value of issue we have found : 4000
2017-06-03 08:48:50	DEBUG	issue we found for is : 004
2017-06-03 08:48:50	DEBUG	integer value of issue we are looking for : 4000
2017-06-03 08:48:50	DEBUG	issue we are looking for is : 4
2017-06-03 08:48:50	DEBUG	this should be a match!
2017-06-03 08:48:50	DEBUG	it's a go captain... - we matched 100.0%!
2017-06-03 08:48:50	DEBUG	Match to series : 100.0 %.
2017-06-03 08:48:50	DEBUG	splitit-len:2
2017-06-03 08:48:50	DEBUG	scount:2
2017-06-03 08:48:50	DEBUG	word matched on : seven
2017-06-03 08:48:50	DEBUG	1 Comparing: seven .to. seven
2017-06-03 08:48:50	DEBUG	scount : 1
2017-06-03 08:48:50	DEBUG	splitit: seven
2017-06-03 08:48:50	DEBUG	word matched on : infinite
2017-06-03 08:48:50	DEBUG	0 Comparing: infinite .to. infinite
2017-06-03 08:48:50	DEBUG	scount : 0
2017-06-03 08:48:50	DEBUG	splitit: infinite
2017-06-03 08:48:50	DEBUG	Watchlist-length: 2
2017-06-03 08:48:50	DEBUG	search-length: 2
2017-06-03 08:48:50	DEBUG	length match..proceeding
2017-06-03 08:48:50	DEBUG	len-watchcomic : 2
2017-06-03 08:48:50	DEBUG	splitst : 2
2017-06-03 08:48:50	DEBUG	['infinite', 'seven'] watchlist word count: 2
2017-06-03 08:48:50	DEBUG	['infinite', 'seven', '004'] nzb series word count: 2
2017-06-03 08:48:50	DEBUG	adjusting from: infinite seven 004 to: 004
2017-06-03 08:48:50	DEBUG	adjusted nzb comic and issue: infinite seven 004
2017-06-03 08:48:50	DEBUG	original nzb comic and issue: infinite seven 004
2017-06-03 08:48:50	DEBUG	Scanner detected: digital dargh-empire
2017-06-03 08:48:50	DEBUG	2. Bracket Word: digital dargh-empire
2017-06-03 08:48:50	DEBUG	2017 - right years match baby!
2017-06-03 08:48:50	DEBUG	year looking for: 2017
2017-06-03 08:48:50	DEBUG	year detected: 2017
2017-06-03 08:48:50	DEBUG	1. Bracket Word: 2017
2017-06-03 08:48:50	DEBUG	Series version detected as V1 (only series in existance with that title). Bypassing Year/Volume check
2017-06-03 08:48:50	DEBUG	ComVersChk : 0
2017-06-03 08:48:50	DEBUG	UseFuzzy is : None
2017-06-03 08:48:50	DEBUG	Comic: infinite seven 004
2017-06-03 08:48:50	DEBUG	0. Bracket Word: infinite seven 004
2017-06-03 08:48:50	DEBUG	Cleantitle: infinite seven 004 (2017) digital dargh-empire
2017-06-03 08:48:50	DEBUG	detected invalid nzb filename - attempting to detect year to continue
2017-06-03 08:48:50	DEBUG	Cleantitle: infinite seven 004 2017 digital dargh-empire
2017-06-03 08:48:50	DEBUG	Entry: Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 08:48:50	DEBUG	Thu, 11 May 2017 13:35:21 -0600 is after store date of 2017-05-10
2017-06-03 08:48:50	DEBUG	sub:Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 08:48:50	DEBUG	checking search result: Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 08:48:50	INFO	no errors on data retrieval...proceeding
2017-06-03 08:48:49	DEBUG	[SSL: True] Search URL: https://api.dognzb.cr/api?t=search&q=infinite%20seven%20004&o=xml&cat=7030&apikey=xUDONTNEEDTOKNOWTHISx
2017-06-03 08:47:49	INFO	pausing for 60 seconds before continuing to avoid hammering
2017-06-03 08:47:49	DEBUG	calpha/cnumber: None / None
2017-06-03 08:47:49	INFO	Shhh be very quiet...I'm looking for Infinite Seven issue: 4 (2017) using dognzb
2017-06-03 08:47:49	DEBUG	Initiating Search via : api
2017-06-03 08:47:49	INFO	search provider order is ['dognzb', 'Experimental']
2017-06-03 08:47:49	DEBUG	provider order sequence is now to start with Experimental at spot #1
2017-06-03 08:47:49	DEBUG	checking against nzb provider: experimental
2017-06-03 08:47:49	DEBUG	checking against nzb provider: dognzb
2017-06-03 08:47:49	DEBUG	found provider in existing enabled providers.
2017-06-03 08:47:49	DEBUG	provider order sequence is now to start with dognzb at spot #0
2017-06-03 08:47:49	DEBUG	checking against nzb provider: dognzb
2017-06-03 08:47:49	DEBUG	found provider in existing enabled providers.
2017-06-03 08:47:49	DEBUG	Usenet Retention : 1500 days
2017-06-03 08:47:49	DEBUG	there are : 2 nzb providers you have selected.
2017-06-03 08:47:49	DEBUG	there are : 0 torrent providers you have selected.
2017-06-03 08:47:49	DEBUG	nzbprovider(s): ['dognzb', 'experimental']
2017-06-03 08:47:49	DEBUG	Checking for torrent enabled.
2017-06-03 08:47:49	INFO	Issue Title given as : None
2017-06-03 08:47:49	DEBUG	Publisher is : Action Lab
2017-06-03 08:47:49	INFO	Initiating manual search for Infinite Seven issue: 4
2017-06-03 08:47:49	DEBUG	mode:want
2017-06-03 08:47:49	DEBUG	ComicID:98499
2017-06-03 08:47:28	DEBUG	If you read this message, debug logging is available
2017-06-03 08:47:28	INFO	Verbose toggled, set to True

NZB Settings within Mylar:
Settings
InformationWeb InterfaceDownload settingsSearch providersQuality & Post ProcessingAdvanced Settings
Usenet
 Sabnzbd   NZBget   Black Hole

NZBGet Host:
http://localhost
usually http://localhost
NZBGet Port:
6789
NZBGet Username:
nzbget
NZBGet Password:
••••••••••
NZBGet Download Directory
Where your NZBGet downloads go... (optional)
NZBGet Category:
Comics
NZBGet Priority

Usenet Retention (in days)
1500
 Torrents

Use Torrents


Local Watch dir

Remote Watch dir
Save Changes 
Web Interface changes require a restart to take effect

**Comment by evilhero** [comment]
Have you tried using 127.0.0.1 instead of localhost for the host address (or even 0.0.0.0)? Depending on the machine and some other factors usually related to cherrypy resolving localhost won't resolve properly to the given machine. Make sure you save & restart mylar after the config change as well. That or the port is being blocked by your firewall possibly (not likely usually but had to be mentioned just in case)

**Comment by evilhero** [comment]
@milkplus27 that sounds more like a post-process issue than what this is related to (actually downloading the file). If mylar and sabnzbd are on different machines than you need to enable the option in the config. If this is something you want me to look into more, please start a new issue and post in the debug logs showing the leadup to the error and try actual error you're receiving and I can help more.

**Comment by aerorush91** [comment]
I have changed the IP to 0.0.0.0 and 127.0.0.1 and now it says I have an invalid host name for NZBGET. Firewalls have been off since I started troubleshooting. FYI.
2017-06-03 11:45:22	INFO	Could not find Issue 3 of Rebels: These Free And Independent States (2017) using dognzb [api]
2017-06-03 11:45:22	ERROR	You have an invalid nzbget hostname specified. Exiting
2017-06-03 11:45:22	DEBUG	Cache Directory successfully found at : c:\Mylar\cache
2017-06-03 11:45:22	DEBUG	[FILENAME] end nzbname: Rebels.-.These.Free.and.Independent.States.003.2017.digital.Son.of.Ultron-Empire.nzb
2017-06-03 11:45:22	DEBUG	[FILENAME] nzbname (\s): Rebels.-.These.Free.and.Independent.States.003.2017.digital.Son.of.Ultron-Empire.nzb
2017-06-03 11:45:22	DEBUG	[FILENAME] filename (remove chars): Rebels.-.These.Free.and.Independent.States.003.2017.digital.Son.of.Ultron-Empire.nzb
2017-06-03 11:45:21	INFO	Download URL: https://dognzb.cr/fetch/f832e541bbfa9c4050773f59f380903b/d13f29b3bc08973f3b97f878923f1259 [VerifySSL:True]
2017-06-03 11:45:21	DEBUG	link given by: dognzb
2017-06-03 11:45:21	INFO	Found Rebels: These Free And Independent States (2017) issue: 3 using dognzb
2017-06-03 11:45:21	DEBUG	issues match!
2017-06-03 11:45:21	DEBUG	nzbname used for post-processing:Rebels.-.These.Free.and.Independent.States.003.2017.digital.Son.of.Ultron-Empire
2017-06-03 11:45:21	DEBUG	[SEARCHER] end nzbname: Rebels.-.These.Free.and.Independent.States.003.2017.digital.Son.of.Ultron-Empire
2017-06-03 11:45:21	DEBUG	[SEARCHER] nzbname (\s): Rebels - These Free and Independent States 003 2017 digital Son of Ultron-Empire
2017-06-03 11:45:21	DEBUG	[SEARCHER] nzbname (remove chars): Rebels.-.These.Free.and.Independent.States.003.2017.digital.Son.of.Ultron-Empire
2017-06-03 11:45:21	DEBUG	[SEARCHER] entry[title]: Rebels.-.These.Free.and.Independent.States.003.2017.digital.Son.of.Ultron-Empire
2017-06-03 11:45:21	DEBUG	integer value of issue we have found : 3000
2017-06-03 11:45:21	DEBUG	issue we found for is : 003
2017-06-03 11:45:21	DEBUG	integer value of issue we are looking for : 3000
2017-06-03 11:45:21	DEBUG	issue we are looking for is : 3
2017-06-03 11:45:21	DEBUG	this should be a match!
2017-06-03 11:45:21	DEBUG	it's a go captain... - we matched 100.0%!
2017-06-03 11:45:21	DEBUG	Match to series : 100.0 %.
2017-06-03 11:45:21	DEBUG	splitit-len:6
2017-06-03 11:45:21	DEBUG	scount:6
2017-06-03 11:45:21	DEBUG	word matched on : states
2017-06-03 11:45:21	DEBUG	5 Comparing: states .to. states
2017-06-03 11:45:21	DEBUG	scount : 5
2017-06-03 11:45:21	DEBUG	splitit: states
2017-06-03 11:45:21	DEBUG	word matched on : independent
2017-06-03 11:45:21	DEBUG	4 Comparing: independent .to. independent
2017-06-03 11:45:21	DEBUG	scount : 4
2017-06-03 11:45:21	DEBUG	splitit: independent
2017-06-03 11:45:21	DEBUG	word matched on : and
2017-06-03 11:45:21	DEBUG	3 Comparing: and .to. and
2017-06-03 11:45:21	DEBUG	scount : 3
2017-06-03 11:45:21	DEBUG	splitit: and
2017-06-03 11:45:21	DEBUG	word matched on : free
2017-06-03 11:45:21	DEBUG	2 Comparing: free .to. free
2017-06-03 11:45:21	DEBUG	scount : 2
2017-06-03 11:45:21	DEBUG	splitit: free
2017-06-03 11:45:21	DEBUG	word matched on : these
2017-06-03 11:45:21	DEBUG	1 Comparing: these .to. these
2017-06-03 11:45:21	DEBUG	scount : 1
2017-06-03 11:45:21	DEBUG	splitit: these
2017-06-03 11:45:21	DEBUG	word matched on : rebels
2017-06-03 11:45:21	DEBUG	0 Comparing: rebels .to. rebels
2017-06-03 11:45:21	DEBUG	scount : 0
2017-06-03 11:45:21	DEBUG	splitit: rebels
2017-06-03 11:45:21	DEBUG	Watchlist-length: 6
2017-06-03 11:45:21	DEBUG	search-length: 6
2017-06-03 11:45:21	DEBUG	length match..proceeding
2017-06-03 11:45:21	DEBUG	len-watchcomic : 6
2017-06-03 11:45:21	DEBUG	splitst : 6
2017-06-03 11:45:21	DEBUG	['rebels', 'these', 'free', 'and', 'independent', 'states'] watchlist word count: 6
2017-06-03 11:45:21	DEBUG	['rebels', 'these', 'free', 'and', 'independent', 'states', '003'] nzb series word count: 6
2017-06-03 11:45:21	DEBUG	adjusting from: rebels these free and independent states 003 to: 003
2017-06-03 11:45:21	DEBUG	adjusted nzb comic and issue: rebels these free and independent states 003
2017-06-03 11:45:21	DEBUG	First word of issue stored as : these
2017-06-03 11:45:21	DEBUG	Assuming position start is : 7
2017-06-03 11:45:21	DEBUG	There is no hyphen present in the series title.
2017-06-03 11:45:21	DEBUG	[Rebels: These Free And Independent States] I have found a - within the nzbname @ position: 7
2017-06-03 11:45:21	DEBUG	original nzb comic and issue: rebels - these free and independent states 003
2017-06-03 11:45:21	DEBUG	Scanner detected: digital son of ultron-empire
2017-06-03 11:45:21	DEBUG	mini-series detected : digital son of ultron-empire
2017-06-03 11:45:21	DEBUG	2. Bracket Word: digital son of ultron-empire
2017-06-03 11:45:21	DEBUG	2017 - right years match baby!
2017-06-03 11:45:21	DEBUG	year looking for: 2017
2017-06-03 11:45:21	DEBUG	year detected: 2017
2017-06-03 11:45:21	DEBUG	1. Bracket Word: 2017
2017-06-03 11:45:21	DEBUG	ComVersChk : 2
2017-06-03 11:45:21	DEBUG	UseFuzzy is : None
2017-06-03 11:45:21	DEBUG	Comic: rebels - these free and independent states 003
2017-06-03 11:45:21	DEBUG	0. Bracket Word: rebels - these free and independent states 003
2017-06-03 11:45:21	DEBUG	Cleantitle: rebels - these free and independent states 003 (2017) digital son of ultron-empire
2017-06-03 11:45:21	DEBUG	detected invalid nzb filename - attempting to detect year to continue
2017-06-03 11:45:21	DEBUG	Cleantitle: rebels - these free and independent states 003 2017 digital son of ultron-empire
2017-06-03 11:45:21	DEBUG	Entry: Rebels.-.These.Free.and.Independent.States.003.2017.digital.Son.of.Ultron-Empire
2017-06-03 11:45:21	DEBUG	Sat, 27 May 2017 12:25:54 -0600 is after store date of 2017-05-24
2017-06-03 11:45:21	DEBUG	sub:Rebels.-.These.Free.and.Independent.States.003.2017.digital.Son.of.Ultron-Empire
2017-06-03 11:45:21	DEBUG	checking search result: Rebels.-.These.Free.and.Independent.States.003.2017.digital.Son.of.Ultron-Empire
2017-06-03 11:45:21	INFO	no errors on data retrieval...proceeding
2017-06-03 11:45:20	DEBUG	[SSL: True] Search URL: https://api.dognzb.cr/api?t=search&q=rebels%20these%20free%20independent%20states%20003&o=xml&cat=7030&apikey=xUDONTNEEDTOKNOWTHISx
2017-06-03 11:44:20	INFO	pausing for 60 seconds before continuing to avoid hammering
2017-06-03 11:44:20	DEBUG	calpha/cnumber: None / None
2017-06-03 11:44:20	INFO	Shhh be very quiet...I'm looking for Rebels These Free And Independent States issue: 3 (2017) using dognzb

**Comment by evilhero** [comment]
You would still have to precede the ip by the protocol (ie. http://127.0.0.1) - what did you have for the host that was giving an invalid error?

**Comment by aerorush91** [comment]
Same thing when 0.0.0.0 or 127.0.0.1
2017-06-03 12:18:54	ERROR	Uncaught exception: Traceback (most recent call last):
File "c:\Mylar\mylar\logger.py", line 158, in new_run
old_run(*args, **kwargs)
File "c:\Python27\lib\threading.py", line 763, in run
self.__target(*self.__args, **self.__kwargs)
File "c:\Mylar\mylar\webserve.py", line 1460, in queueissue
foundcom, prov = search.search_init(ComicName, ComicIssue, ComicYear, SeriesYear, Publisher, issues['IssueDate'], storedate, IssueID, AlternateSearch, UseAFuzzy, ComicVersion, mode=mode, ComicID=ComicID, manualsearch=manualsearch, filesafe=ComicName_Filesafe, allow_packs=AllowPacks)
File "c:\Mylar\mylar\search.py", line 260, in search_init
findit = NZB_SEARCH(ComicName, IssueNumber, ComicYear, SeriesYear, Publisher, IssueDate, StoreDate, searchprov, send_prov_count, IssDateFix, IssueID, UseFuzzy, newznab_host, ComicVersion=ComicVersion, SARC=SARC, IssueArcID=IssueArcID, RSS="no", ComicID=ComicID, issuetitle=issuetitle, unaltered_ComicName=unaltered_ComicName, allow_packs=allow_packs)
File "c:\Mylar\mylar\search.py", line 1609, in NZB_SEARCH
searchresult = searcher(nzbprov, nzbname, comicinfo, entry['link'], IssueID, ComicID, tmpprov, newznab=newznab_host)
File "c:\Mylar\mylar\search.py", line 2350, in searcher
send_to_nzbget = server.append(nzbpath, str(mylar.NZBGET_CATEGORY), int(nzbgetpriority), True, nzbcontent64)
File "c:\Python27\lib\xmlrpclib.py", line 1233, in __call__
return self.__send(self.__name, args)
File "c:\Python27\lib\xmlrpclib.py", line 1591, in __request
verbose=self.__verbose
File "c:\Python27\lib\xmlrpclib.py", line 1273, in request
return self.single_request(host, handler, request_body, verbose)
File "c:\Python27\lib\xmlrpclib.py", line 1301, in single_request
self.send_content(h, request_body)
File "c:\Python27\lib\xmlrpclib.py", line 1448, in send_content
connection.endheaders(request_body)
File "c:\Python27\lib\httplib.py", line 997, in endheaders
self._send_output(message_body)
File "c:\Python27\lib\httplib.py", line 850, in _send_output
self.send(msg)
File "c:\Python27\lib\httplib.py", line 812, in send
self.connect()
File "c:\Python27\lib\httplib.py", line 793, in connect
self.timeout, self.source_address)
File "c:\Python27\lib\socket.py", line 571, in create_connection
raise err
error: [Errno 10049] The requested address is not valid in its context
2017-06-03 12:18:54	DEBUG	Cache Directory successfully found at : c:\Mylar\cache
2017-06-03 12:18:54	DEBUG	[FILENAME] end nzbname: Infinite.Seven.004.2017.digital.dargh-Empire.nzb
2017-06-03 12:18:54	DEBUG	[FILENAME] nzbname (\s): Infinite.Seven.004.2017.digital.dargh-Empire.nzb
2017-06-03 12:18:54	DEBUG	[FILENAME] filename (remove chars): Infinite.Seven.004.2017.digital.dargh-Empire.nzb
2017-06-03 12:18:53	INFO	Download URL: https://dognzb.cr/fetch/c2b144020030b71f711819cac49cbad2/d13f29b3bc08973f3b97f878923f1259 [VerifySSL:True]
2017-06-03 12:18:53	DEBUG	link given by: dognzb
2017-06-03 12:18:53	INFO	Found Infinite Seven (2017) issue: 4 using dognzb
2017-06-03 12:18:53	DEBUG	issues match!
2017-06-03 12:18:53	DEBUG	nzbname used for post-processing:Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:18:53	DEBUG	[SEARCHER] end nzbname: Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:18:53	DEBUG	[SEARCHER] nzbname (\s): Infinite Seven 004 2017 digital dargh-Empire
2017-06-03 12:18:53	DEBUG	[SEARCHER] nzbname (remove chars): Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:18:53	DEBUG	[SEARCHER] entry[title]: Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:18:53	DEBUG	integer value of issue we have found : 4000
2017-06-03 12:18:53	DEBUG	issue we found for is : 004
2017-06-03 12:18:53	DEBUG	integer value of issue we are looking for : 4000
2017-06-03 12:18:53	DEBUG	issue we are looking for is : 4
2017-06-03 12:18:53	DEBUG	this should be a match!
2017-06-03 12:18:53	DEBUG	it's a go captain... - we matched 100.0%!
2017-06-03 12:18:53	DEBUG	Match to series : 100.0 %.
2017-06-03 12:18:53	DEBUG	splitit-len:2
2017-06-03 12:18:53	DEBUG	scount:2
2017-06-03 12:18:53	DEBUG	word matched on : seven
2017-06-03 12:18:53	DEBUG	1 Comparing: seven .to. seven
2017-06-03 12:18:53	DEBUG	scount : 1
2017-06-03 12:18:53	DEBUG	splitit: seven
2017-06-03 12:18:53	DEBUG	word matched on : infinite
2017-06-03 12:18:53	DEBUG	0 Comparing: infinite .to. infinite
2017-06-03 12:18:53	DEBUG	scount : 0
2017-06-03 12:18:53	DEBUG	splitit: infinite
2017-06-03 12:18:53	DEBUG	Watchlist-length: 2
2017-06-03 12:18:53	DEBUG	search-length: 2
2017-06-03 12:18:53	DEBUG	length match..proceeding
2017-06-03 12:18:53	DEBUG	len-watchcomic : 2
2017-06-03 12:18:53	DEBUG	splitst : 2
2017-06-03 12:18:53	DEBUG	['infinite', 'seven'] watchlist word count: 2
2017-06-03 12:18:53	DEBUG	['infinite', 'seven', '004'] nzb series word count: 2
2017-06-03 12:18:53	DEBUG	adjusting from: infinite seven 004 to: 004
2017-06-03 12:18:53	DEBUG	adjusted nzb comic and issue: infinite seven 004
2017-06-03 12:18:53	DEBUG	original nzb comic and issue: infinite seven 004
2017-06-03 12:18:53	DEBUG	Scanner detected: digital dargh-empire
2017-06-03 12:18:53	DEBUG	2. Bracket Word: digital dargh-empire
2017-06-03 12:18:53	DEBUG	2017 - right years match baby!
2017-06-03 12:18:53	DEBUG	year looking for: 2017
2017-06-03 12:18:53	DEBUG	year detected: 2017
2017-06-03 12:18:53	DEBUG	1. Bracket Word: 2017
2017-06-03 12:18:53	DEBUG	Series version detected as V1 (only series in existance with that title). Bypassing Year/Volume check
2017-06-03 12:18:53	DEBUG	ComVersChk : 0
2017-06-03 12:18:53	DEBUG	UseFuzzy is : None
2017-06-03 12:18:53	DEBUG	Comic: infinite seven 004
2017-06-03 12:18:53	DEBUG	0. Bracket Word: infinite seven 004
2017-06-03 12:18:53	DEBUG	Cleantitle: infinite seven 004 (2017) digital dargh-empire
2017-06-03 12:18:53	DEBUG	detected invalid nzb filename - attempting to detect year to continue
2017-06-03 12:18:53	DEBUG	Cleantitle: infinite seven 004 2017 digital dargh-empire
2017-06-03 12:18:53	DEBUG	Entry: Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:18:53	DEBUG	Thu, 11 May 2017 13:35:21 -0600 is after store date of 2017-05-10
2017-06-03 12:18:53	DEBUG	sub:Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:18:53	DEBUG	checking search result: Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:18:53	INFO	no errors on data retrieval...proceeding
2017-06-03 12:18:53	DEBUG	[SSL: True] Search URL: https://api.dognzb.cr/api?t=search&q=infinite%20seven%20004&o=xml&cat=7030&apikey=xUDONTNEEDTOKNOWTHISx
2017-06-03 12:17:57	DEBUG	If you read this message, debug logging is available
2017-06-03 12:17:57	INFO	Verbose toggled, set to True
2017-06-03 12:17:53	INFO	pausing for 60 seconds before continuing to avoid hammering
2017-06-03 12:17:53	INFO	Shhh be very quiet...I'm looking for Infinite Seven issue: 4 (2017) using dognzb
2017-06-03 12:17:53	INFO	search provider order is ['dognzb', 'Experimental']
2017-06-03 12:17:53	INFO	Issue Title given as : None
2017-06-03 12:17:53	INFO	Initiating manual search for Infinite Seven issue: 4

**Comment by aerorush91** [comment]
Not the same thing...

**Comment by aerorush91** [comment]
When I run with 0.0.0.0 that is the above log, when I run it with 127.0.0.1 or localhost I get this.


Timestamp	Level	Message
2017-06-03 12:25:50	ERROR	Uncaught exception: Traceback (most recent call last):
File "c:\Mylar\mylar\logger.py", line 158, in new_run
old_run(*args, **kwargs)
File "c:\Python27\lib\threading.py", line 763, in run
self.__target(*self.__args, **self.__kwargs)
File "c:\Mylar\mylar\webserve.py", line 1460, in queueissue
foundcom, prov = search.search_init(ComicName, ComicIssue, ComicYear, SeriesYear, Publisher, issues['IssueDate'], storedate, IssueID, AlternateSearch, UseAFuzzy, ComicVersion, mode=mode, ComicID=ComicID, manualsearch=manualsearch, filesafe=ComicName_Filesafe, allow_packs=AllowPacks)
File "c:\Mylar\mylar\search.py", line 260, in search_init
findit = NZB_SEARCH(ComicName, IssueNumber, ComicYear, SeriesYear, Publisher, IssueDate, StoreDate, searchprov, send_prov_count, IssDateFix, IssueID, UseFuzzy, newznab_host, ComicVersion=ComicVersion, SARC=SARC, IssueArcID=IssueArcID, RSS="no", ComicID=ComicID, issuetitle=issuetitle, unaltered_ComicName=unaltered_ComicName, allow_packs=allow_packs)
File "c:\Mylar\mylar\search.py", line 1609, in NZB_SEARCH
searchresult = searcher(nzbprov, nzbname, comicinfo, entry['link'], IssueID, ComicID, tmpprov, newznab=newznab_host)
File "c:\Mylar\mylar\search.py", line 2350, in searcher
send_to_nzbget = server.append(nzbpath, str(mylar.NZBGET_CATEGORY), int(nzbgetpriority), True, nzbcontent64)
File "c:\Python27\lib\xmlrpclib.py", line 1233, in __call__
return self.__send(self.__name, args)
File "c:\Python27\lib\xmlrpclib.py", line 1591, in __request
verbose=self.__verbose
File "c:\Python27\lib\xmlrpclib.py", line 1273, in request
return self.single_request(host, handler, request_body, verbose)
File "c:\Python27\lib\xmlrpclib.py", line 1301, in single_request
self.send_content(h, request_body)
File "c:\Python27\lib\xmlrpclib.py", line 1448, in send_content
connection.endheaders(request_body)
File "c:\Python27\lib\httplib.py", line 997, in endheaders
self._send_output(message_body)
File "c:\Python27\lib\httplib.py", line 850, in _send_output
self.send(msg)
File "c:\Python27\lib\httplib.py", line 812, in send
self.connect()
File "c:\Python27\lib\httplib.py", line 793, in connect
self.timeout, self.source_address)
File "c:\Python27\lib\socket.py", line 571, in create_connection
raise err
error: [Errno 10061] No connection could be made because the target machine actively refused it
2017-06-03 12:25:49	DEBUG	Cache Directory successfully found at : c:\Mylar\cache
2017-06-03 12:25:49	DEBUG	[FILENAME] end nzbname: Infinite.Seven.004.2017.digital.dargh-Empire.nzb
2017-06-03 12:25:49	DEBUG	[FILENAME] nzbname (\s): Infinite.Seven.004.2017.digital.dargh-Empire.nzb
2017-06-03 12:25:49	DEBUG	[FILENAME] filename (remove chars): Infinite.Seven.004.2017.digital.dargh-Empire.nzb
2017-06-03 12:25:47	INFO	Download URL: https://dognzb.cr/fetch/c2b144020030b71f711819cac49cbad2/d13f29b3bc08973f3b97f878923f1259 [VerifySSL:True]
2017-06-03 12:25:47	DEBUG	link given by: dognzb
2017-06-03 12:25:47	INFO	Found Infinite Seven (2017) issue: 4 using dognzb
2017-06-03 12:25:47	DEBUG	issues match!
2017-06-03 12:25:47	DEBUG	nzbname used for post-processing:Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:25:47	DEBUG	[SEARCHER] end nzbname: Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:25:47	DEBUG	[SEARCHER] nzbname (\s): Infinite Seven 004 2017 digital dargh-Empire
2017-06-03 12:25:47	DEBUG	[SEARCHER] nzbname (remove chars): Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:25:47	DEBUG	[SEARCHER] entry[title]: Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:25:47	DEBUG	integer value of issue we have found : 4000
2017-06-03 12:25:47	DEBUG	issue we found for is : 004
2017-06-03 12:25:47	DEBUG	integer value of issue we are looking for : 4000
2017-06-03 12:25:47	DEBUG	issue we are looking for is : 4
2017-06-03 12:25:47	DEBUG	this should be a match!
2017-06-03 12:25:47	DEBUG	it's a go captain... - we matched 100.0%!
2017-06-03 12:25:47	DEBUG	Match to series : 100.0 %.
2017-06-03 12:25:47	DEBUG	splitit-len:2
2017-06-03 12:25:47	DEBUG	scount:2
2017-06-03 12:25:47	DEBUG	word matched on : seven
2017-06-03 12:25:47	DEBUG	1 Comparing: seven .to. seven
2017-06-03 12:25:47	DEBUG	scount : 1
2017-06-03 12:25:47	DEBUG	splitit: seven
2017-06-03 12:25:47	DEBUG	word matched on : infinite
2017-06-03 12:25:47	DEBUG	0 Comparing: infinite .to. infinite
2017-06-03 12:25:47	DEBUG	scount : 0
2017-06-03 12:25:47	DEBUG	splitit: infinite
2017-06-03 12:25:47	DEBUG	Watchlist-length: 2
2017-06-03 12:25:47	DEBUG	search-length: 2
2017-06-03 12:25:47	DEBUG	length match..proceeding
2017-06-03 12:25:47	DEBUG	len-watchcomic : 2
2017-06-03 12:25:47	DEBUG	splitst : 2
2017-06-03 12:25:47	DEBUG	['infinite', 'seven'] watchlist word count: 2
2017-06-03 12:25:47	DEBUG	['infinite', 'seven', '004'] nzb series word count: 2
2017-06-03 12:25:47	DEBUG	adjusting from: infinite seven 004 to: 004
2017-06-03 12:25:47	DEBUG	adjusted nzb comic and issue: infinite seven 004
2017-06-03 12:25:47	DEBUG	original nzb comic and issue: infinite seven 004
2017-06-03 12:25:47	DEBUG	Scanner detected: digital dargh-empire
2017-06-03 12:25:47	DEBUG	2. Bracket Word: digital dargh-empire
2017-06-03 12:25:47	DEBUG	2017 - right years match baby!
2017-06-03 12:25:47	DEBUG	year looking for: 2017
2017-06-03 12:25:47	DEBUG	year detected: 2017
2017-06-03 12:25:47	DEBUG	1. Bracket Word: 2017
2017-06-03 12:25:47	DEBUG	Series version detected as V1 (only series in existance with that title). Bypassing Year/Volume check
2017-06-03 12:25:47	DEBUG	ComVersChk : 0
2017-06-03 12:25:47	DEBUG	UseFuzzy is : None
2017-06-03 12:25:47	DEBUG	Comic: infinite seven 004
2017-06-03 12:25:47	DEBUG	0. Bracket Word: infinite seven 004
2017-06-03 12:25:47	DEBUG	Cleantitle: infinite seven 004 (2017) digital dargh-empire
2017-06-03 12:25:47	DEBUG	detected invalid nzb filename - attempting to detect year to continue
2017-06-03 12:25:47	DEBUG	Cleantitle: infinite seven 004 2017 digital dargh-empire
2017-06-03 12:25:47	DEBUG	Entry: Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:25:47	DEBUG	Thu, 11 May 2017 13:35:21 -0600 is after store date of 2017-05-10
2017-06-03 12:25:47	DEBUG	sub:Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:25:47	DEBUG	checking search result: Infinite.Seven.004.2017.digital.dargh-Empire
2017-06-03 12:25:47	INFO	no errors on data retrieval...proceeding
2017-06-03 12:25:46	DEBUG	[SSL: True] Search URL: https://api.dognzb.cr/api?t=search&q=infinite%20seven%20004&o=xml&cat=7030&apikey=xUDONTNEEDTOKNOWTHISx
2017-06-03 12:24:50	DEBUG	If you read this message, debug logging is available
2017-06-03 12:24:50	INFO	Verbose toggled, set to True
2017-06-03 12:24:46	INFO	pausing for 60 seconds before continuing to avoid hammering
2017-06-03 12:24:46	INFO	Shhh be very quiet...I'm looking for Infinite Seven issue: 4 (2017) using dognzb

My Control IP on NZBGet is set for a different address for login that shouldn't make a difference. The set  up on NZBGet hasn't changed, but Mylar for some reason cannot connect to it.

**Comment by evilhero** [comment]
Well if you set the control ip to something other than localhost/127.0.0.1 then it will only listen for connections on that specific interface. The only catch being that setting it to 0.0.0.0 will make nzbget listen to all interfaces. So it actually has alot to do with your connection problems. 

Mylar is trying to connect to localhost or 127.0.0.1 and nzbget is listening in an exact ip, neither of which is those 2 addresses. Either change the controlip in nzbget to 0.0.0.0 / 127.0.0.1 /localhost, or specify the nzbget controlip nzbget is running under in the mylar host address field.

**Comment by aerorush91** [comment]
Ok it's working now every time I would change it to the control IP NZB has set Mylar would say I don't have a search provider listed. What you said about restarting after a change worked. Thank you for you help. I am still a noob at figuring this stuff out.
